rust/stackable-cockpit/src/xfer/cache.rs

```diff
@@ -20,12 +25,19 @@ pub enum Error {
     #[snafu(display("system time error"))]
     SystemTimeError { source: SystemTimeError },
 
+    #[snafu(display("failed to parse string as integer"))]
+    ParseIntError { source: ParseIntError },
```

Comment:
There error messages are nigh-useless, _what_ are you trying to parse? Where should the user look to diagnose the issue?

Reply:
Agreed. Tbh I was just a little lazy to come up with a better message :)

Reply:
Removed, not needed anymore. 2eaaa8b